### PR TITLE
Update license

### PR DESCRIPTION
**Pull request by iamnandhu**
removed some lines from the license so that it meets 98% (95 before) string match to bsd-3-clause

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,7 @@
 BSD 3-Clause License
 
-British Crown copyright. The Met Office.
-All rights reserved.
+Copyright (c) 2022, The Met Office.
+git
```
Comment (cpelley) [inline LICENSE:4]:
```suggestion
```
Does the standard github BSD 3-Clause License add "git" like this here?

**Comment by cpelley** [comment]
Thanks for the contribution @iamnandhu 
I suspect this PR relates to the issue https://github.com/metoppv/improver/issues/1776
Is that right?

If so, I see the motivation would be in reaching a 95% threshold with [licensee](https://github.com/licensee/licensee#the-solution).  Awesome.

@Kat-90, you recently removed dates from the copyright after consulting our legal dep.
Do you know if the proposed change to the copyright (removal of 'British Crown copyright' in particular) is within parameters discussed with legal?

Cheers

**Comment by iamnandhu** [comment]
> Thanks for the contribution @iamnandhu I suspect this PR relates to the issue #1776 Is that right?
> 
> If so, I see the motivation would be in reaching a 95% threshold with [licensee](https://github.com/licensee/licensee#the-solution). Awesome.
> 
> @Kat-90, you recently [removed dates from the copyright](https://github.com/metoppv/improver/pull/1716) after consulting our legal dep. Do you know if the proposed change to the copyright (removal of 'British Crown copyright' in particular) is within parameters discussed with legal?
> 
> Cheers
When including 'British crown copyright's and 'All rights reserved' the license file had 95.2% similarity to bsd-3-clause
Required is 98% to auto detect.

File: LICENSE (hunk 1 above)
Comment (iamnandhu) [reply LICENSE:4]: Sorry!🙆 My mistake
Let me resolve it

**Comment by Kat-90** [comment]
@cpelley No this is not within the parameters discussed with legal. We will need to have further discussion before anything goes ahead.

**Comment by cpelley** [comment]
Running `licensee` locally I see that it is sensitive to identifying attribution by the inclusion of the substituted [copyright symbol](https://en.wikipedia.org/wiki/Copyright_symbol) `(c)`.
This means that we can minimally change our copyright notice to adhere to a 100% match:

```diff
-  British Crown copyright. The Met Office.
+ (c) British Crown copyright. The Met Office.
```

@benfitzpatrick  - Given this, I don't think we need to follow-up with legal after all.  To give extra confidence, I have run through some other MO repositories, where some also include the copyright symbol so we are all good.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -73,6 +73,7 @@ below:
  - Tomasz Trzeciak (Met Office, UK)
  - Mark Worsfold (Met Office, UK)
  - Ying Zhao (Bureau of Meteorology, Australia)
+ - Nandhu S (Student, India)
```
Comment (benfitzpatrick) [inline CONTRIBUTING.md:76]: @iamnandhu, thank you! Would you mind moving your name higher so that the surname (S) is alphabetically sorted?

**Comment by benfitzpatrick** [comment]
OK brilliant. Let's make that `(c)` change then

File: LICENSE (hunk 1 above)
Comment (cpelley) [inline LICENSE:4]:
```suggestion
(c) British Crown copyright. The Met Office.
```

**Review by cpelley — CHANGES_REQUESTED** [review]

**Comment by cpelley** [comment]
Thanks for looking at this 👍 
Closed in favour of https://github.com/metoppv/improver/pull/1785

**Pull request closed without merging**